apps/src/code-studio/pd/application_dashboard/constants.js

```diff
@@ -60,3 +60,14 @@ exports.ApplicationStatuses = {
     'Withdrawn'
   ]
 };
+
+/**
+ * Constants for Regional Partner dropdown
+ */
+const allPartners = "All Regional Partner Applications";
+exports.AllPartners = allPartners;
```

Comment:
Any reason for the mismatched casing between the local and export?

Reply:
I capitalized the export because it is an export and that seems to be the convention, and for the local I didn't because that also seems to be convention...but I don't feel strongly about the capitalization if you would rather it be the same.

Reply:
Makes sense. I probably would name the local all upper case like a CONSTANT, but this is fine too.